Source/loadsave.cpp

```diff
@@ -337,6 +338,7 @@ static void LoadPlayer(LoadHelper *file, int p)
 	file->skip(4); // Unused
 	player._pgfxnum = file->nextLE<int32_t>();
 	file->skip(4); // Skip pointer pData
+	player.AnimInfo = {};

```

Comment:
This explains the weird behavior @qndel mentioned not being able to repro with the character, right?

Reply:
I can check if this fixes it for me as well.

Reply:
Fixed my issues